students/uss_tomasz/lesson_05_lists/5_5/numbers_filter.py

```diff
@@ -0,0 +1,22 @@
+# Problem - Numbers filter
+def strings_list_to_integers_list(strings_list_from_user, filter_range):
```

Comment:
Same as above - this function should not care if the list is from user or any other source.